nuxt.config.ts

```diff
@@ -18,8 +17,13 @@ export default defineNuxtConfig({
   ],
   nitro: {
     prerender: {
-      crawlLinks: false
-    }
+      crawlLinks: true,
+      routes: ['/en/', '/nl/'],
+    },
+  },
+  routeRules: {
+    '/**': { swr: true },
```

Comment:
How do the routeRules work combined with the prerender options? I think it's good to put in some comments about that.

Reply:
Think the Nuxt docs should explain this in their docs.